internal/resolution/resolver.go

```diff
@@ -7,9 +7,8 @@ import (
 	"github.com/operator-framework/deppy/pkg/deppy/solver"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	"github.com/operator-framework/operator-controller/internal/resolution/variable_sources/olm"
-
 	"github.com/operator-framework/operator-controller/api/v1alpha1"
+	"github.com/operator-framework/operator-controller/internal/resolution/variable_sources/olm"
```

Comment:
I strongly recommend using https://github.com/openshift-eng/openshift-goimports in CI

Reply:
This definitely is a larger conversation to bring up in the upstream community meetings so that it can be discussed and endorsed by the community first

Reply:
Sure - the tool runs in a couple milliseconds so I don't imagine it will be a hard thing to convince, but definitely out of scope for this PR

Reply:
It isn't so much about the run time, than it is about "we're about to use an openshift product for this project". We've been having conversations about getting away from open shift native products for our projects (prow/openshift-ci etc) in favor of more cncf native projects, so this feels like a good item to include in that list too.

Reply:
Just because something exists under an openshift repo doesn't mean its a problem. If it solves a problem and doesn't give RH or RH-products preferential treatment, it's fine.

There are also other reasonable upstream tools if we're averse to that for some reason. I've used https://github.com/rinchsan/gosimports

Reply:
Yeah it's a super trivial utility .... is OpenShift is that much of a dirty word?